### PR TITLE
test: add integration tests for claiming trapped asset

### PR DESCRIPTION
**Pull request by enthusiastmartin**

### CODE REVIEW

File: integration-tests/src/cross_chain_transfer.rs [hunk 1]
```diff
@@ -487,3 +487,202 @@ fn assets_should_be_trapped_when_assets_are_unknown() {
...
+			399_600_000_000
+		));
+		assert_eq!(
+			basilisk_runtime::Balances::free_balance(&AccountId::from(BOB)),
+			1000 * UNITS - 30 * UNITS
+		);
+		assert_eq!(
+			basilisk_runtime::Tokens::free_balance(1, &basilisk_runtime::Treasury::account_id()),
+			35_990_141 // fees should go to treasury
+		);
```
Comment (apopiak) [inline integration-tests/src/cross_chain_transfer.rs:571]: why not check that they arrive back on the Hydra chain?

File: integration-tests/src/cross_chain_transfer.rs [hunk 2]
```diff
@@ -487,3 +487,202 @@ fn assets_should_be_trapped_when_assets_are_unknown() {
...
+		]);
+		let origin = MultiLocation::new(1, X1(Parachain(3000)));
+		let hash = determine_hash(&origin, vec![asset.clone()]);
+		assert_eq!(basilisk_runtime::PolkadotXcm::asset_trap(hash), 1);
+	});
+
+	asset
+}
+
+use polkadot_xcm::VersionedXcm;
```
Comment (apopiak) [inline integration-tests/src/cross_chain_transfer.rs:626]: nit: probably move it up to the top of the file?

**Review by apopiak — APPROVED** [review]
LGTM if you add a check that the transfer back was received

**Review by lumir-mrkva — APPROVED** [review]
LGTM after clippy stops hating you

File: integration-tests/src/cross_chain_transfer.rs (hunk 1 above)
Comment (enthusiastmartin) [reply integration-tests/src/cross_chain_transfer.rs]: i am not sure anymore why  this test was there at all as part of this PR. i removed it. this scenarios are already covered by other testcases.

**Pull request merged**